ludwig/schema/llms/base_model.py

```diff
@@ -0,0 +1,102 @@
+from dataclasses import field
+
+from marshmallow import fields, ValidationError
+from transformers import AutoConfig
+
+from ludwig.api_annotations import DeveloperAPI
+from ludwig.constants import BASE_MODEL
+from ludwig.error import ConfigValidationError
+from ludwig.schema.metadata import LLM_METADATA
+from ludwig.schema.metadata.parameter_metadata import convert_metadata_to_json
+
+MODEL_PRESETS = {
```

Comment:
nit: Can we document this global variable?

> Maps a preset LLM name to the full slash-delimited HF path. If the user chooses a preset LLM, the preset LLM name is replaced with the full slash-delimited HF path using this map, after JSON validation but before config object initialization.

Reply:
done